pkg/dinghyfile/builder.go

```diff
@@ -538,3 +538,16 @@ func getParams(regEx, url string) (paramsMap map[string]string) {
 	}
 	return paramsMap
 }
+
+func (b *PipelineBuilder) getContent() map[string]interface{} {
+	content := map[string]interface{}{}
```

Comment:
double {}

Reply:
I think it's fine actually, this reads like "instantiate a map of string to interface"